chore(docs) Remove instances of let's to resolve mdx lint warnings (#47013)
supabase/supabase: apps/docs/content/guides/ai/examples/nextjs-vector-search.mdx

    Anytime the user sends a query, we want to find the content that's relevant to their questions. We can do this using pgvector's similarity search.

>     These are quite complex SQL operations, wrap them in database functions that you can call from the frontend using [RPC](/docs/reference/javascript/rpc).

Author: ```suggestion
    For complex SQL operations, wrap them in database functions that you can call from the frontend using [RPC](/docs/reference/javascript/rpc).
```
